kolibri/plugins/coach/assets/src/views/home/HomePage/OverviewBlock.vue

```diff
@@ -120,7 +120,7 @@
         context: 'Refers to the learner or learners who are in a class.',
       },
       viewLearners: {
-        message: 'View learners',
+        message: 'View learner devices',
```

Comment:
The other update that @pcenov was asking for here was to make the display of the link conditional on this page, in the same way it is in the ReportsControls component.